apps/ejabberd/src/mod_mam_odbc_async_pool_writer.erl

```diff
@@ -60,6 +60,13 @@ worker_prefix() ->
 worker_count(_Host) ->
```

Comment:
There is function [gen_mod:get_module_opt/4](https://github.com/esl/MongooseIM/blob/d17b636c02c77bfd65e74bc212f73dbf8f931a98/apps/ejabberd/src/gen_mod.erl#L40).